crates/qasm3/src/circuit.rs

```diff
@@ -235,9 +235,8 @@ impl PyCircuitModule {
     ) -> PyResult<PyQuantumRegister> {
         let qreg = self.qreg.call1(py, (size, name))?;
         Ok(PyQuantumRegister {
-            items: qreg
-                .bind(py)
-                .getattr("_bits")?
+            items: PyList::type_object(py)
+                .call1((qreg.clone(),))?
```

Comment:
Can't you just use the pyo3 constructor for a `PyList` here? Something like:
```rust
items: PyList::new(py, qreg.clone())?.unbind(),
```

Reply:
Since these still use the Register types from Python, I believe this should be the one way to use it. I can modify it to use the rust types in the future.